api/api/templates/admin/api/media_report/change_form.html

```diff
@@ -0,0 +1,159 @@
+  {% if media_type == 'image' %}
+    <div class="img-container">
+      <img src="{% url media_type|add:'-thumb' identifier=identifier %}" alt="Media Image" class="blur-image" height="300"
+           onclick="toggleBlur(this)"
+           onerror="this.onerror=null;this.src='{{ url }}';"
+           style="cursor: pointer;">
+      <p>Show content</p>
+    </div>
+
+  {% elif media_type == 'audio' %}
+    <audio controls>
+      <source src="{{ url }}" type="audio/mpeg">
+      Your browser does not support the audio element.
+    </audio>
+  {% endif %}
+
+  <div class="form-row field-created_at">
+      <div class="flex-container">
+        <label>Title and creator</label>
+        <div class="readonly">
+          <p>{{ title }} {% if creator_url %}by <a href="{{creator_url}}">{{creator}}</a> {% else %} {{creator}} {% endif %} </p></div>
+      </div>
+  </div>
+  <div class="form-row field-created_at">
+      <div class="flex-container">
+        <label>Media links</label>
+        <div class="readonly">
+          <p><a href="{{foreign_landing_url}}">Source ({{source}})</a></p>
+          <p><a href="https://openverse.org/{{media_type}}/{{identifier}}">openverse.org</a></p>
+          <p><a href="{% url media_type|add:'-detail' identifier=identifier %}">openverse API</a></p>
+        </div>
+      </div>
+  </div>
+  <div class="form-row field-created_at">
+      <div class="flex-container">
+        <label>Tags</label>
+        <div class="readonly">
+          {% for provider, provider_tags in tags.items %}
+            <p><em>{{ provider }}</em></p>
+            <p> {% for tag in provider_tags %}{{ tag }}, {% endfor %}</p>
+          {% endfor %}
+        </div>
+      </div>
+  </div>
+  <div class="form-row field-created_at">
+      <div class="flex-container">
+        <label>Description</label>
+        <div class="readonly">
+          <p>{{ description }}</div>
+      </div>
+  </div>
+  <div class="form-row field-created_at">
+      <div class="flex-container">
+        <label>From</label>
+        <div class="readonly">
+          <p>provider: {{ provider }}, source: {{ source }}</div>
+      </div>
+  </div>
+</fieldset>
+
+<fieldset class="module aligned">
+  <h2>Other reports</h2>
+  <div class="form-row field-other_reports">
+  {% if other_reports %}
+    <table>
+      <thead>
+        <tr>
+          <th>Date</th>
+          <th>Report reason</th>
+          <th>Status</th>
+          <th>Report link</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for report in other_reports %}
+        <tr>
+          <td>{{ report.created_at }}</td>
+          <td>{{ report.reason }}</td>
+          <td>{{ report.status }}</td>
+          <td><a href="{% url 'admin:api_'|add:media_type|add:'report_change' report.id %}">{{ report.id }}</a></td>
+        </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+  {% else %}
+    <p>No other reports for <a href="{{reverse_media_url}}">{{ identifier }}</a> found.</p>
```

Comment:
When I click on this for a report, I get redirected to the admin home page with this message 🤔 

![image](https://github.com/WordPress/openverse/assets/10214785/5dcd999f-7d65-40f3-b46c-ea14b407f4ad)

The URL it's trying to go to is something like http://localhost:50280/admin/api/image/de37812d-2daa-49f6-91ab-e87e780ca653/change/, is it possibly because the `/change/` URL isn't available for images? Should something else be used here instead?

Reply:
That'll be the database `id`, primary key column.